docs/guides/operator/rolling-updates.adoc

```diff
@@ -71,6 +71,15 @@ When using the `Auto` update strategy, the {project_name} Operator automatically
 Read more about the process in the <@links.server id="update-compatibility"/> {section}.
 This process consumes cluster resources for the time of the check and introduces a slight delay before the StatefulSet update begins.
 
+[WARNING]
+====
+If the Keycloak CR configured a `podTemplate` as part of the `unsupported` configuration parameters, the Keycloak Operator will do its best to use those settings for the started Job. Still it might miss some settings due to the flexibility of the `podTemplate` feature and its unsupported nature.
+
+As a consequence, the Operator might draw the wrong conclusions if a rolling update is possible from changes to the `podTemplate` or information pulled in from Secrets, ConfigMaps or Volumes in the `podTemplate`.
+
+Therefore, the recommended way is to run the `Auto` command without setting the `podTemplate`.
```

Comment:
Should the recommendation be phrased more like:

Therefore if you are using the unsupported `podTemplate`, you may need to use one of the other update strategies.

Reply:
Thank you for the suggestion. I updated the PR as you suggested.